server/config.js

```diff
@@ -11,8 +11,8 @@ const schema = joi.object({
   serviceUrl: joi.string().uri().default('http://localhost:8050'),
   geoserverUrl: joi.string().uri().default('http://localhost:8080'),
   bingKey: joi.string().required(),
-  bingUrl: joi.string().uri(),
-  nrwStationUrl: joi.string().uri(),
+  bingUrl: joi.string().required(),
```

Comment:
You took out the `uri()` part but I think this should remain for the vars so `joi.string().uri().required()` should work fine? @JFSCH

Reply:
Change made as requested.